environments/staging-aws-test/meta.yml

```diff
@@ -0,0 +1,7 @@
+deploy_env: staging
+env_monitoring_id: staging-aws-test
+users:
+#  - dimagi
+#  - dimagi-qa
+  - droberts
+  - g2
```

Comment:
`g2` can be removed.

Reply:
Yeah I'll check with Lucina and remove from all environments once they're done